packages/manager/modules/carrier-sip/src/endpoints/endpoints.html

```diff
@@ -1,37 +1,44 @@
 <section>
-    <h3 data-translate="carrier_sip_endpoints"></h3>
-    <oui-datagrid data-rows="$ctrl.endpoints">
+    <h3 data-translate="carrier_sip_endpoints_list_IP"></h3>
+    <oui-datagrid class="w-25" rows="$ctrl.endpointIpList">
```

Comment:
Please let me know if it could suit your needs because using the `w-25` helper from small devices won't work efficiently.

```suggestion
    <oui-datagrid rows="$ctrl.endpointIpList">
```

Reply:
Hello @antleblanc , I'm curious about what won't work efficiently about this class helepr ? (I've check on Bootstrap https://getbootstrap.com/docs/4.0/utilities/sizing/ and it seems to work correctly)

Reply:
`w-25` in portrait mode can lead to a non-stretched datagrid.

Reply:
Component class has been added to add min-width property to this element